### PR TITLE
Corrected Powershell profiles location and added test for the same.

### LINKED ISSUE #1190: New PowerShell Profiles for PowerShell
# Overview
PowerShell Core instances shall support their own set of profiles. The profiles shall be placed in  PowerShell Core specific directories and have PowerShell Core specific names. Additional code will be needed to load the profiles from the new locations. Mostly it looks like a task to IFDEF the components of the name and path to be appropriate for PowerShell Core.

# New Profiles
Name = PowerShell Core File and path
AllUsersAllHosts = $PSHome\profile.ps1
AllUsersCurrentHost = $PSHome\Microsoft.PowerShell_profile.ps1
CurrentUserAllHosts = ~\Documents\PowerShell\profile.ps1
CurrentUserCurrentHost = ~\Documents\PowerShell\Microsoft.PowerShell_profile.ps1

### PR DESCRIPTION
**Pull request by charub**
Resolve #1190 
For windows :
AllUsersAllHosts = $PSHome\profile.ps1
AllUsersCurrentHost = $PSHome\PowerShell_profile.ps1
CurrentUserAllHosts = ~\Documents\PowerShell\profile.ps1
CurrentUserCurrentHost = ~\Documents\PowerShell\PowerShell_profile.ps1

For Linux :
AllUsersAllHosts = $PSHOME/profile.ps1
AAllUsersCurrentHost = $PSHOME/PowerShell_profile.ps1
PCurrentUserCurrentHost =
/home/user/.config/powershell/PowerShell_profile.ps1
$CurrentUserAllHosts = /home/user/.config/powershell/profile.ps1

### CODE REVIEW

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs [hunk 1]
```diff
@@ -1724,8 +1724,11 @@ private void DoRunspaceInitialization(bool importSystemModules, bool skipProfile
                 if (_configuration != null)
                     shellId = _configuration.ShellId;
                 else
+#if CORECLR                
+                    shellId = Platform.IsInbox ?  "Microsoft.PowerShell" : "PowerShell";
+#else
```
Comment (daxian-dbw) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs]: I'm not sure about the necessity to make this change. I just put a comment at #1190 for @mirichmo to review and response.

Comment (daxian-dbw) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs]: @charub the discussion conclusion is that we don't need to change the profile file names for powershell core. Please see the comments in #1190 for details.
Give the discussion conclusion, do you think we need to change anything to resolve issue #1190 ?

**Comment by daxian-dbw** [comment]
@charub the discussion conclusion is that we don't need to change the profile file names for powershell core. Please see the comments in #1190 for details.
Give the discussion conclusion, do you think we need to change anything to resolve issue #1190 ?

**Comment by charub** [comment]
@daxian-dbw : As per the discussion we want to keep the same profile file names. So this change is no longer required.

**Pull request closed without merging**